pkg/alertmanager/operator.go

```diff
@@ -266,17 +265,6 @@ func (c *Operator) enqueue(obj interface{}) {
 	c.queue.Add(key)
 }
 
-// enqueueForNamespace enqueues all Alertmanager object keys that belong to the
-// given namespace.
-func (c *Operator) enqueueForNamespace(ns string) {
```

Comment:
Is this just not used anywhere?

Reply:
yes :) I guess it was copied/pasted at some point from prometheus.